api/client-server/login.yaml

```
@@ -194,10 +194,13 @@ paths:
             "$ref": "definitions/errors/error.yaml"
         403:
           description: |-
-            The login attempt failed. For example, the password may have been incorrect.
+            The login attempt failed. This may include one of the following error codes:
```

we don't use may if we can avoid it.
```suggestion
            The login attempt failed. This can include one of the following error codes:
```

---

This was copied from https://matrix.org/docs/spec/client_server/unstable#post-matrix-client-r0-register btw